ts/torch_handler/image_classifier.py

```diff
@@ -4,6 +4,10 @@
 import torch
 import torch.nn.functional as F
 from torchvision import transforms
+from torchvision import models
```

Comment:
@abishekchiffon Why do need this in the base image handler?

Reply:
We don't need it, Geeta. Lines 7, 9, 10 will be removed.